spec/webdriver_spec.vim

```diff
@@ -0,0 +1,28 @@
+source spec/support/helpers.vim
+
+describe "Webdriver"
+
+  before
+    cd spec/fixtures/webdriver
+  end
+
+  after
+    call Teardown()
+    cd -
+  end
+
+  it "runs file tests"
+    view test/specs/example.js
+    TestFile
+
+    Expect g:test#last_command == 'wdio wdio.conf.js ---spec test/specs/example.js'
+  end
+
```

Comment:
Let's also add a test where we test that `:TestNearest` executes a file test (same as `:TestFile`).